synopsys-task/src/synopsys-task/synopsys-bridge.ts

```diff
@@ -469,4 +520,18 @@ export class SynopsysBridge {
     }
     return bridgeDownloadUrl;
   }
+
+  async setBridgeExecutablePath(
```

Comment:
we are setting executable path when airgap is false, you can use this method there as well in executeBridgeCommand method